pkg/controller/stacks/install/installjob.go

```diff
@@ -40,6 +40,13 @@ import (
 	"github.com/crossplaneio/crossplane/pkg/stacks"
 )
 
+// Labels used to track ownership across namespaces and scopes.
+const (
+	labelInstallNamespace = "stacks.crossplane.io/install-namespace"
+	labelInstallName      = "stacks.crossplane.io/install-name"
+	labelInstallUID       = "stacks.crossplane.io/install-uid"
```

Comment:
Just wondering if we really need UIDs? I know owner references uses them but isn't name and namespace enough?

Reply:
Namespace and name are only unique at a particular point in time; UID is necessary to identify the owning object over time. In this case the UID is an extra layer of protection ensuring that the delete logic only cleans up CRDs that were really installed by this `StackInstall`, not a previous `StackInstall` with the same namespace and name that didn't clean up after itself.

It also allows us to use the same label set for `StackInstall` and `ClusterStackInstall`, given one of each could exist with the same namespace and name at the same time. We could also address this by adding a type metadata label.